app/services/RegistrationService.scala

```diff
@@ -71,7 +71,7 @@ class RegistrationService @Inject()(
         option: Option[DetailedIndividualAccount],
         addressId: Long,
         groupAccount: Option[GroupAccount],
-        affinityGroupOpt: Option[AffinityGroup] = None)(
+        affinityGroupOpt: Option[AffinityGroup])(
```

Comment:
just double-checking you are sure about removing the default (None)?

Reply:
of course, scala 2.12 can spot when the default value is never used. all callers explicitly provide this argument. neat